docs/snippets/rst/button-link.txt

```diff
@@ -1,3 +1,8 @@
+.. _buttons:
+
+Buttons
+.......
+
```

Comment:
Ditto for this. Added to make tests pass. Patterned after the [card-link snippet](https://github.com/executablebooks/sphinx-design/blob/main/docs/snippets/rst/card-link.txt)